src/DotNetEnv/LoadOptions.cs

```diff
@@ -4,47 +4,54 @@ namespace DotNetEnv
 {
     public class LoadOptions
     {
-        public static readonly LoadOptions DEFAULT = new LoadOptions();
-
-        public bool SetEnvVars { get; }
-        public bool ClobberExistingVars { get; }
-        public bool OnlyExactPath { get; }
+        public bool SetEnvVars { get; set; }
+        public bool ClobberExistingVars { get; set; }
+        public bool OnlyExactPath { get; set; }
 
         public LoadOptions(
             bool setEnvVars = true,
             bool clobberExistingVars = true,
             bool onlyExactPath = true
-        ) {
+        )
+        {
             SetEnvVars = setEnvVars;
             ClobberExistingVars = clobberExistingVars;
             OnlyExactPath = onlyExactPath;
         }
 
-        public LoadOptions(
-            LoadOptions old,
-            bool? setEnvVars = null,
-            bool? clobberExistingVars = null,
-            bool? onlyExactPath = null
-        ) {
-            SetEnvVars = setEnvVars ?? old.SetEnvVars;
-            ClobberExistingVars = clobberExistingVars ?? old.ClobberExistingVars;
-            OnlyExactPath = onlyExactPath ?? old.OnlyExactPath;
-        }
+        public static LoadOptions DEFAULT => new LoadOptions();
```

Comment:
Before `DEFAULT` returned a readonly static instance. Now it calls a function every time it is referenced, returning a mutable value, which is not an improvement.

Reply:
Static instance is not an option without Immutability of course.

So this depends on how we/you decide on the relevance of the added features/possibilities.

If the decision will be to favor immutability, I will "clean up" everything, just making the code more precise, but keeping Immutability-things as they were before.